doc/related_projects.rst

```diff
@@ -27,6 +27,9 @@ enhance the functionality of scikit-learn's estimators.
 - `sklearn-pmml <https://github.com/alex-pirozhenko/sklearn-pmml>`_
   Serialization of (some) scikit-learn estimators into PMML.
 
+- `sklearn2pmml <https://github.com/jpmml/sklearn2pmml>`_
+  Serialization of scikit-learn estimators and transformers in PMML data format.
```

Comment:
The `sklearn2pmml` project can fully supports more than 20 estimator types (most popular clustering, classification amd regression types) and close to 10 transformer types. As such, it is *actually useful* for someone who is trying to migrate their SkLearn workflows to PMML.

The `sklearn-pmml` project, on the other hand, claims to support three estimator types (with limitations).